main.py

```diff
@@ -1,7 +1,32 @@
+from contextlib import asynccontextmanager
+
 from fastapi import FastAPI
+from starlette.middleware.cors import CORSMiddleware
+
+from auth.config import get_settings
 from routers import user
+from routers import galaxy_register
```

Comment:
```suggestion
from routers import galaxy_register, user
```
merge the two imports